### DISCUSSION TITLE
I can't add a column in board view.

### DISCUSSION THREAD

**Discussion by ahhhhhh100** [marcusolsson/obsidian-projects#865] [Help] [answered]
When I type in the name of the new column and press enter or click off, the column goes away.

**Comment by ghost** [comment]
I have the same problem

**Comment by Acylation** [comment]
Oh then this should be considered a bug. Can you post more details to help me reproduce this?

You may include
- The project config, i.e. data source, nested tag/subfolder
- The status field in the problem board, and its correponding field config (can access from table view, column details)
- Your file structure / tag structure if the subfolders / nested tags are tracked
- OS, and Obsidian app version
- Example problematic new column names, and existing columns

Please note that duplicated columns are omitted by default so you will fail create one in this case.

Thank you so much!

**Comment by msmart** [comment, marked as answer]
To me this happens when the Status is set to "None".  To resolve this you need to set the status to a value
![image](https://github.com/marcusolsson/obsidian-projects/assets/85249/45b6d444-e281-4bea-a6ff-085e66ccfd6c)

**Comment by Netajam** [comment]
If the status is using a field of type number we cannot add a new column.

**Reply by Acylation** [reply]
Sorry but I can't reproduce. Please check if the field is a read-only one or you're under a read-only project. Recently I'll publish 1.17.4 including some fixes to the board and please feel free to reach back if the problem persists.
